src/main/webapp/app/core/config/datepicker-adapter.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Angular bootstrap Date adapter
+ */
+import { Injectable } from '@angular/core';
+import { NgbDateAdapter, NgbDateStruct } from '@ng-bootstrap/ng-bootstrap';
+import dayjs from 'dayjs';
+
+@Injectable()
+export class NgbDateDayjsAdapter extends NgbDateAdapter<dayjs.Dayjs> {
+    fromModel(date: dayjs.Dayjs | null): NgbDateStruct | null {
+        if (date && dayjs.isDayjs(date) && date.isValid()) {
```

Comment:
Seems so. I would argue to leave it like this, as it does not break anything.

Reply:
```suggestion
        if (date && date.isValid()) {
```
Why though? Leaving things in the code that is not needed just because "it does not break anything" seems not like a worthwhile approach in coding.

I'd make this suggestion instead.